fuel-moshi/build.gradle.kts

```diff
@@ -0,0 +1,5 @@
+plugins { java }
+dependencies {
```

Comment:
[Nits] Can you add extra new line between `plugins` and `dependencies` like counterparts?

Reply:
Changed in b6392d8.